event.go

```diff
@@ -34,17 +40,20 @@ func CreateListener(channel, query string, startpos EVT_SUBSCRIBE_FLAGS, watcher
 	if err != nil {
 		return 0, err
 	}
-	listenerHandle, err := EvtSubscribe(0, 0, wideChan, wideQuery, 0, uintptr(unsafe.Pointer(watcher)), uintptr(syscall.NewCallback(eventCallback)), uint32(startpos))
+	listenerHandle, err := EvtSubscribe(0, 0, wideChan, wideQuery, 0, uintptr(0), syscall.NewCallback(newEventCallback(watcher)), uint32(startpos))
```

Comment:
Just a note about the `syscall.NewCallback` usage:

> Only a limited number of callbacks may be created in a single Go process, and any memory allocated for these callbacks is never released. Between NewCallback and NewCallbackCDecl, at least 1024 callbacks can always be created.

If these functions could be called repeatedly or for a large number of channels, it might be safer to refactor in a way that minimizes the number of callbacks created.